docs/setup.md

```diff
@@ -0,0 +1,105 @@
+## Setting up the system
+
+We assume cuda drivers and [cuDNN](https://developer.nvidia.com/cudnn) is installed. cuda installation instructions are [here](https://askubuntu.com/questions/799184/how-can-i-install-cuda-on-ubuntu-16-04). cuDNN installation instructions are [described below](#cudnn-installation).  
+
+### Conda
+
+For maintaining a virtual environment in python, we use `conda`. You can use other package managers such as `pip`. Conda can be installed by [downloading this file](https://repo.continuum.io/archive/Anaconda3-4.4.0-Linux-x86_64.sh) and using `bash <installer.sh>`. Download the latest version from [this link](https://www.continuum.io/downloads).
```

Comment:
Change first 3 sentences to "if you want to use `conda`, you can install it..."  There is no couple to any ecosystem, so no need to recommend anything